### PR TITLE
HeatmapLayer/GPUAggregator: fix WebGL feature checking

### PR DESCRIPTION
**Pull request by 1chandu**
For #3480 

Requires: https://github.com/uber/luma.gl/pull/1210

#### Background
Fix `feature` checking in Heatmap Layer and GPUGridAggregator.

#### Change List
- HeatmapLayer/GPUAggregator: fix WebGL feature checking

### CODE REVIEW

File: modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js [hunk 1]
```diff
@@ -55,13 +55,26 @@ const defaultProps = {
...
+
 export default class HeatmapLayer extends CompositeLayer {
   initializeState() {
     const {gl} = this.context;
     const textureSize = Math.min(SIZE_2K, getParameter(gl, gl.MAX_TEXTURE_SIZE));
     this.state = {textureSize, supported: true};
-    if (!isWebGL2(gl)) {
-      log.error(`HeatmapLayer ${this.id} is not supported on this browser, requires WebGL2`)();
+    if (!hasFeatures(gl, REQUIRED_FEATURES)) {
+      const info = REQUIRED_FEATURES.reduce(
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js]: Call `GPUAggregator.isSupported`?

File: modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js [hunk 2]
```diff
@@ -55,13 +55,26 @@ const defaultProps = {
...
+        (acc, feature) =>
+          `${acc} ${feature}: ${hasFeatures(gl, feature) ? 'SUPPORTED' : 'NOT SUPPORTED'} `,
```
Comment (Pessimistress) [inline modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js]: - this is technically debug info. I suggest moving this into `hasFeature` and use `log.log(NON_ZERO_PRIORITY, ...)`
- `map` and then `join` will be more readable.

File: modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js (hunk 1 above)
Comment (1chandu) [reply modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js]: Though the checks are same, HeatmapLayer doesn't use GPUGridAggregator, in future it is possible these feature list will change.

**Comment by 1chandu** [comment]
luma.gl now logs, when feature is checked and not supported. Updated `HeatmapLayer` to just log single statement when not supported.

**Pull request merged**